matlab_simulation/code v1.0/ME597-4-GyroCovariance/gyroCovEstimation.m

```diff
@@ -0,0 +1,90 @@
+%% A Matlab program to calculate the convariances from a Gyroscope Data.
```

Comment:
This file looks identical to the one above under the directory "02-examples_practice/gyroCovEstimation/". Only one copy of the file should exist, perhaps this one should be removed.